Vic2ToHoI4/Source/OutHoi4/OutHoi4World.cpp

```diff
@@ -400,6 +400,17 @@ void HoI4::outputCountries(const std::set<Advisor>& activeIdeologicalAdvisors,
 	}
 	ideasFile << "\n";
 	ideasFile << "}\n";
+	ideasFile.close();
+
+	std::ofstream portraitsFile("output/" + outputName + "/portraits/conv_portraits.txt");
+	for (const auto& country: countries)
```

Comment:
if (!portraitsFile.is_open())
throw std::runtime_error("Could not open output/" + outputName + "/interface/conv_portraits.gfx");